docs/git.md

```diff
@@ -9,10 +9,55 @@ At Matomo we're using Git version control system. Below are some tips for using
 
 ### Undo last commit that is not pushed yet without losing the changes
 
-`git reset --soft HEAD~1`
+```git reset --soft HEAD~1```
```

Comment:
btw for one liner this is not needed a single tick works fine/the same

Reply:
fyi @geekdenz I will keep it as it is. Next time better to just leave the single backtick